src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+import UserService from '../services/userServices';
+import Util from '../utils/response';
+import Helper from '../middlewares/index';
+const { findUserById, updateUser } = UserService;
+const util = new Util();
+
+/** Class representing user controller. */
+class UserController {
+  /**
+  * get user profile
+  * @param {Object} req - server request
+  * @param {Object} res - server response
+  * @returns {Object} - custom response
+  * @description get details of registered user
+  */
+  static async getUserProfile(req, res) {
+    const { id } = req.params;
+    try {
+      const user = await findUserById(id);
+      const token = Helper.generateToken(id);
+      console.log(token)
```

Comment:
Remove console statement

Reply:
It is used to generate token

Reply:
I don't understand

Reply:
I understand now. Instead of logging the token in the console, why not send it as response in the get route?

Reply:
Oh
That's true 
Thank You